### PR TITLE
feat(PaperMC): add presence

### PR DESCRIPTION
**Pull request by BrycensRanch**
I created a Presence for PaperMC and HangarMC because I saw Modrinth had one and Spigot did, but Paper didn't. Here's a showcase of it in action! https://imgur.com/a/vRMgcjd

### Acknowledgements
 I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
 I linted the code by running `yarn format`
 The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

### CODE REVIEW

**Comment by BrycensRanch** [comment]
why are the checks failing?? I've followed all the guidelines 😭

**Comment by theusaf** [comment]
> why are the checks failing?? I've followed all the guidelines 😭

It looks like you accidentally committed files not related to your presence:
* `package-lock.json`
* `websites/V/Video Game Mods`

Simply remove these, and the validations should pass.

File: websites/P/PaperMC/metadata.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "387062216030945281",
+		"name": "Romvnly#6466"
```
Comment (theusaf) [inline websites/P/PaperMC/metadata.json:5]:
```suggestion
		"name": "Romvnly"
```

File: websites/P/PaperMC/metadata.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	"url": [
+		"papermc.io",
+		"docs.papermc.io",
+		"jd.papermc.io",
+		"hangar.papermc.io",
+		"hangar.papermc.dev",
+		"forums.papermc.io"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/SzuS0AE.png",
```
Comment (theusaf) [inline websites/P/PaperMC/metadata.json:20]: Logos should be [`512x512px`](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo)

**Comment by BrycensRanch** [comment]
are you sure? github check annotations says I didn't change the file but there's a issue with another presence I didnttouch

**Comment by BrycensRanch** [comment]
oh. that stupid format command it ran format on all the presences

**Comment by BrycensRanch** [comment]
how do I untouch a file?

File: websites/P/PaperMC/metadata.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	"thumbnail": "https://i.imgur.com/pPFKTFD.jpeg",
```
Comment (theusaf) [inline websites/P/PaperMC/metadata.json:21]: Higher-res version
```suggestion
	"thumbnail": "https://i.imgur.com/Dlt8uSV.jpg",
```

File: websites/P/PaperMC/presence.ts [hunk 4]
```diff
@@ -0,0 +1,287 @@
+const presence = new Presence({
+		clientId: "1088185685682430002",
+	}),
+	logoKey = "papermc-logomark-500-b69f67cabd469b3d0485c20a912e84fc_1_",
```
Comment (theusaf) [inline websites/P/PaperMC/presence.ts:4]: Imgur links are used over Discord asset names in this repository.

**Comment by BrycensRanch** [comment]
fuck it

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/P/PaperMC/metadata.json (hunk 3 above)
Comment (BrycensRanch) [reply websites/P/PaperMC/metadata.json:21]: are you sure? this thumbnail is kinda watermarked with "PAPER" in the middle ruining the graphic in general

**Comment by theusaf** [comment]
To revert changes to a file, you can use things such as `git checkout` or `git reset`. You can also rebase and modify the commit that added the unwanted changes, then force push.

File: websites/P/PaperMC/metadata.json (hunk 3 above)
Comment (theusaf) [reply websites/P/PaperMC/metadata.json:21]: Ah, I didn't see that. Well, there should be a higher-resolution version out there somewhere.

Comment (theusaf) [reply websites/P/PaperMC/metadata.json:21]: Here's a better one
```suggestion
	"thumbnail": "https://i.imgur.com/0TPvY1q.png",
```

**Comment by BrycensRanch** [comment]
watch the MAGIC happen

**Comment by BrycensRanch** [comment]
im calm.

**Comment by BrycensRanch** [comment]
image
I'm currently running into issues implementing your suggestion, @theusaf . Doesn't pass Discord's regex

**Comment by BrycensRanch** [comment]
Oh, I see what you mean now.

**Review by theusaf — APPROVED** [review]
LGTM

**Comment by BrycensRanch** [comment]
Any tips on how to fix my commit signatures? My original commits were all signed.

**Comment by theusaf** [comment]
> Any tips on how to fix my commit signatures? My original commits were all signed.

I wouldn't worry about it that much here. Your commits get automatically squashed into a single non-signed commit when this PR is merged. If you still want to, I found this [superuser question](https://superuser.com/a/1123928/1602946) that could help.

**Comment by BrycensRanch** [comment]
Thanks, thank you for the support!

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by BrycensRanch** [comment]
the other committer is me too I just have a messed up git config for gitpod

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**